src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -118,5 +121,21 @@ void Navigate(Frame frame)
 				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
+		{
+			var appTilesPath = Path.Combine(Package.Current.InstalledLocation.Path, "Assets/AppTiles");
+
+			if (Directory.Exists(Path.Combine(appTilesPath, "Dev")))
+				return Path.Combine(appTilesPath, "Dev", "Logo.ico");
```

Comment:
Also what about extracting `Logo.ico` in a constant stored inside the `Constant.cs`?

Reply:
It doesn't change much. To be seen in the previous point.